web_quick_start_screen/README.rst

```diff
@@ -0,0 +1,110 @@
+==================
+Quick Start Screen
+==================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:f9c2d78e553eef639adf52a6811c9a92a1b131a13c2882469d1e47f4fe271679
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/16.0/web_quick_start_screen
+    :alt: OCA/web
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_quick_start_screen
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/web&target_branch=16.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module adds a configurable start screen so users can go right into
+their more common tasks.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Use Cases / Context
+===================
+
+Some users use repeatedly the same actions on and on and sometimes those
+actions are so many clicks away which tends to be frustrating.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+-  Go to *Settings > Technical > User Interface > Screen actions* and
+   create o modify an existing one.
+-  Choose the desired action, a name, a description and an icon.
+-  Once save, it will show up in the quick start screen.
+
+Usage
+=====
+
+Once you login, you'll go to the quick actions screen. Just click in and
```

Comment:
This shouldn't be something out of the box for all the users. I think this module should provide an action that can be put as initial action in the users configuration to go to the screen. Other options:

- A security group to include users in.
- A check in the user.